docs/users_guide/landice/test_groups/ismip6_forcing.rst

```diff
@@ -0,0 +1,235 @@
+.. _landice_ismip6_forcing:
+
+ismip6_forcing
+============
+
+The ``landice/ismip6_forcing`` test group processes (i.e., remaps and renames)
+the atmospheric and ocean forcing data of the Ice Sheet Model Intercomparison for CMIP6
+(ISMIP6) protocol. The processed data is used to force MALI in future ice sheet
+simulations. The test group includes three test cases, ``atmosphere``,``ocean_basal`` and
+``ocean_thermal``. Each test case contains one main step, called
+``ProcessSMB``, ``ProcessBasalMelt`` and ``ProcessThermalForcing``.
+Each step has the local methods (functions) of remapping and
+renaming the original ismip6 data to the format that MALI can incorporate in
+its forward simulations. In remapping the data, all test cases import the
+method ``build_mapping_file`` to create or use scrip files
+of the source (ISMIP6) and destination (MALI) mesh depending on the existence
+of a mapping file. Approximated time for processing a single forcing file
+on Cori (single core) is 2, 7, and 1 minutes for the atmosphere, ocean basal,
+and ocean thermal testcase, respectively.
+
+The ISMIP6 source data can be obtained by directly
+contacting the authors. Reference: https://www.climate-cryosphere.org/wiki/index.php?title=ISMIP6-Projections2300-Antarctica#A2.2_Retrieving_datasets_and_uploading_your_model_output
```

Comment:
```suggestion
contacting the authors. `Reference <https://www.climate-cryosphere.org/wiki/index.php?title=ISMIP6-Projections2300-Antarctica#A2.2_Retrieving_datasets_and_uploading_your_model_output>`_
```

Reply:
I don't think it will work to put in the full URL and anyway I'd prefer not to.

Reply:
Putting in the URL does seem to work but I still would prefer to have it be a link like above.

Reply:
Also, rather than
```
The ISMIP6 source data can be obtained by directly
contacting the authors
```
I would say something like, "The ISMIP6 source data can be obtained by contacting the ISMIP6 steering committee as described..."

Reply:
The steering committee aren't necessarily the authors.